strictdoc/backend/sdoc_source_code/models/source_file_info.py

```diff
@@ -69,6 +69,14 @@ def __init__(self, g_parts: List[Any]):  # noqa: ARG002
 
         self.markers: List[RelationMarkerType] = []
 
+        self.file_bytes: bytes = b""
```

Comment:
So we cache all source file content as permanent part of the traceability index now, right? Problems:
- Caches get outdated. What if a file changes on disk after we have read it?
- For linux, `find . -type f \( -name "*.c" -o -name "*.h" \) -print0 | du -sch --files0-from=-` reports 1,5G which will have to be pickled and have to be placed all at once in memory (per child).

Could it be an approach to turn `self.file_bytes` into a property that reads the file content temporary on demand?